docs/admin/addons.rst

```diff
@@ -133,6 +124,13 @@ from where you can check whether the configuration actually matches your needs:
 .. seealso::
 
     :ref:`markup`
+    
```

Comment:
Don't know if this is the way to do things, but I had great difficulty figuring out what this was, and still don't understand it being where it was …?

Reply:
fixed :) Thank you!